cattle/plugins/docker/compute.py

```diff
@@ -763,6 +765,25 @@ def _setup_hostname(self, create_config, instance):
         except (KeyError, AttributeError):
             pass
 
+    def _setup_iops(self, config, instance):
+        log.info('setting up iops for docker container named: %s',
+                 instance.name)
+        try:
+            for readIops in instance.data['fields']['blkioDeviceReadIOps']:
+                readIopsDic = {}
+                readIopsDic['Path'] = readIops['Path']
+                readIopsDic['Rate'] = int(readIops['Rate'])
+                log.info("appending readIops: %s", str(readIopsDic))
+                config['BlkioDeviceReadIOps'] = [readIopsDic]
```

Comment:
nice catch, I was testing one item. This line "config['BlkioDeviceReadIOps'] = [readIopsDic]" needs to be append. (my log already saying that, but code does not do). I will change this.